### PR TITLE
Make address display wider in Account Details

### PR DESCRIPTION
**Pull request by whymarrh**
This PR makes the address display wider on the Account Details page.

**Before & after:**

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]

File: ui/app/css/itcss/components/sections.scss [hunk 1]
```diff
@@ -403,10 +403,10 @@ textarea.twelve-word-phrase {
 }
 
 .qr-ellip-address, .ellip-address {
-  /*rtl:ignore*/
```
Comment (Gudahtt) [inline ui/app/css/itcss/components/sections.scss:406]: Why was this removed? I think this is still required to prevent the direction of the address from being flipped when using a RTL locale

Comment (whymarrh) [reply ui/app/css/itcss/components/sections.scss:406]: Is... is this comment used? 🤦

Comment (Gudahtt) [reply ui/app/css/itcss/components/sections.scss]: Yes - the `rtl` comments are directives for `rtlcss`, which we use via `gulp-rtlcss`

Comment (whymarrh) [reply ui/app/css/itcss/components/sections.scss]: I've added it back, that's unfortunate

**Review by Gudahtt — APPROVED** [review]
LGTM!
